tests/vufind.php-cs-fixer.php

```diff
@@ -45,7 +54,10 @@
     'no_useless_return' => true,
     'no_whitespace_in_blank_line' => true,
     'non_printable_character' => true,
-    'ordered_imports' => true,
+    'ordered_imports' => [
```

Comment:
You changed ordered_imports here, but not in the template configuration below. I realize that this is probably because the setting is not particularly relevant to templates... but maybe it's better to make the configurations as consistent as possible so this doesn't confuse us in future if we compare the two files. Or is there a reason they need to be different?